src/main/java/com/epam/prejap/teatrees/game/Playfield.java

```diff
@@ -105,6 +100,10 @@ private void forEachBrick(BrickAction action) {
         }
     }
 
+    public void removeCompleteLines() {
```

Comment:
public APIs should be described, using javadocs. We suggest you keeping that in mind going forward.

Reply:
Done.